packages/trader/src/App/Components/Elements/VideoStream/video-overlay.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+type TVideoOverlay = {
+    onClick: (e: React.MouseEvent<HTMLElement, MouseEvent> | React.KeyboardEvent<HTMLElement>) => void;
+    is_ended?: boolean;
+    is_mobile?: boolean;
+};
+
+const VideoOverlay = ({ onClick, is_ended, is_mobile }: TVideoOverlay) => {
+    return (
+        <div
+            className='player__overlay'
+            style={{ opacity: `${is_ended ? '1' : '0'}` }}
```

Comment:
Need to change opacity (not hiding or not displaying) because on onClick function. User should be able to pause/play video by clicking on the screen

Reply:
let's not use inline styles but classnames instead :) same for all inline styles in this PR